.travis.yml

```diff
@@ -2,14 +2,20 @@ language: c++
 language: python
 python:
   - "3.7"
+dist: bionic
 
-before_install:
-  - curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
-  - echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
-  - sudo apt-get update -qq
+addons:
+  apt:
+    sources:
+      - sourceline: 'deb https://dl.yarnpkg.com/debian/ stable main'
+        key_url: 'https://dl.yarnpkg.com/debian/pubkey.gpg'
+    packages:
+      - ocaml
+      - ocamlbuild
+      - texlive-full
+      - yarn
 
 install:
-  - ./interpreter/meta/travis/install-ocaml.sh
```

Comment:
Looks like this could be removed now.

Reply:
The `install-ocaml.sh` isn't needed anymore?

Reply:
Yea it can be removed. I use case is when a developer does not have ocaml 4.05.0 from their package repo, they can use the script to download and build ocaml.

Reply:
OK, makes sense.